socorro/processor/mozilla_transform_rules.py

```diff
@@ -1116,6 +1116,9 @@ def version(self):
 
     #--------------------------------------------------------------------------
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
+        if not processed_crash.get('addons'):
```

Comment:
I see what you're after here. I'll merge this but alter the implementation of predicate a little.